catroid/src/main/java/org/catrobat/catroid/content/actions/StitchAction.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2018 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.content.actions;
+
+import android.graphics.PointF;
+
+import com.badlogic.gdx.scenes.scene2d.actions.TemporalAction;
+
+import org.catrobat.catroid.content.Sprite;
+import org.catrobat.catroid.stage.StageActivity;
+
+public class StitchAction extends TemporalAction {
+
+	private Sprite sprite;
+
+	@Override
+	protected void update(float delta) {
```

Comment:
I can't see where the delta is used. Is it a relict from overriding? Is there a way to get rid of this?

Reply:
yeah this parameter comes from overriding the update function, can't really see a way to get around having this parameter. (a lot of actions ignore it like i do ^^)